app/controllers/profiles_controller.rb

```diff
@@ -207,8 +203,8 @@ def profiles_for_category
   def profiles_for_search
     Profile.is_published
            .includes(:taggings, :translations)
-           .search(params[:search], params[:filter_countries], params[:filter_cities], params[:filter_lang])
-           .page(params[:page]).per(24)
+           .pagy_search(params[:search])
```

Comment:
@rebecarancan the problem you are encountering might be based on that line here. The `search` method expects 4 arguments `(params[:search], params[:filter_countries], params[:filter_cities], params[:filter_lang])` but you are only providing the first one. So I think adding them might solve the problem.